android/java/res/layout/fragment_connect_account.xml

```diff
@@ -0,0 +1,122 @@
+        tools:text="Site permissions" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="@color/wallet_edittext_outline_color"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:id="@+id/favicon"
+            android:layout_height="120dp"
+            android:layout_width="120dp"
+            android:padding="16dp"
+            android:contentDescription="@null"
+            android:visibility="gone"/>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/fragment_connect_account_website"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="2dp"
+                android:layout_marginTop="6dp"
+                android:layout_marginBottom="4dp"
+                android:padding="2dp"
+                android:gravity="start"
+                android:textColor="@color/brave_wallet_dapp_text_color"
+                android:textSize="14sp"
+                tools:text="Web site" />
+
+            <TextView
+                android:id="@+id/fragment_connect_account_accounts_connected"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="2dp"
+                android:layout_marginTop="2dp"
+                android:layout_marginBottom="4dp"
+                android:padding="2dp"
+                android:gravity="start"
+                android:textColor="@color/brave_wallet_dapp_text_color"
+                android:textSize="14sp"
+                tools:text="Accounts connected" />
+
+        </LinearLayout>
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/fragment_connect_account_new_account_id"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/fragment_connect_account_new_account"
+        android:padding="16dp"
+        android:clickable="true"
+        android:focusable="true"
+        android:textColor="@color/modern_blue_300"
+        android:gravity="start"/>
+
+     <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="16dp"
+        android:paddingHorizontal="16dp"
+        android:orientation="vertical">
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_alignParentTop="true"
+            android:background="@color/wallet_edittext_outline_color" />
+
+        <ScrollView
```

Comment:
do we need ScrollView here? recyclerview scroll can also work here

Reply:
I removed it. It works fine without it. I don't recall why did I put it there originally.